packages/react/ds/src/label/label.tsx

```diff
@@ -8,7 +8,7 @@ export enum LabelSize {
 
 // Extend `React.LabelHTMLAttributes<HTMLLabelElement>` for correct label attributes
 export type LabelProps = React.LabelHTMLAttributes<HTMLLabelElement> & {
-  text: string;
+  text: ReactNode;
```

Comment:
Change this to accept things like: <strong>Bold text</strong> like macro does.

Reply:
Not sure if we want to do this, it will give the user the ability to add any components which will result in unexpected results

Reply:
agreed, changing back to string

Reply:
yes, I was thinking the same.